geomesa-gs-styling/src/main/scala/org/geomesa/gs/styling/filter/GeoMesaFastMark.scala

```diff
@@ -20,7 +22,8 @@ class GeoMesaFastMark extends FunctionExpressionImpl(
 
   override def evaluate(o: java.lang.Object): AnyRef = {
     val icon = getExpression(0).evaluate(o).asInstanceOf[java.lang.String]
-    val rotation: java.lang.Double = Option(getExpression(1).evaluate(o).asInstanceOf[java.lang.Double]).getOrElse(0.0d)
+    val inputRot = getExpression(1).evaluate(o).toString
```

Comment:
instead of toString, use the built in geotools Conversions class:
```
Option(org.geotools.util.Converters.convert(getExpression(1).evaluate(o), classOf[java.lang.Float])).map(_.floatValue).getOrElse(0f)
```